docs/tutorial/using-selenium-and-webdriver.md

```diff
@@ -140,23 +142,19 @@ const options = {
   desiredCapabilities: {
     browserName: 'chrome',
     'goog:chromeOptions': {
-      binary: '/Path-to-Your-App/electron', // Path to your Electron binary.
-      args: [/* cli arguments */] // Optional, perhaps 'app=' + /path/to/your/app/
+      binary: './node_modules/electron/dist/electron',
```

Comment:
```suggestion
      binary: require('electron'),
```

The `electron` module simply exports the path to the binary so this is a simple way of making it work